### ISSUE TITLE
Cannot focus "add row" button

### ISSUE THREAD

**Issue by barmac** [bpmn-io/dmn-js#846]
### Describe the Bug

The "add row" button (+) is inaccessible with the keyboard as it's a `span` without `tabindex`.

### Steps to Reproduce

Try to focus it.

### Expected Behavior

It should be a `button` with accessible label.

### Environment

Please complete the following information:

 - Browser: [e.g. IE 11, Chrome 69]
 - OS: [e.g. Windows 7]
 - Library version: [e.g. 2.0.0]

---

Child of https://github.com/camunda/product-hub/issues/2189

**Comment by barmac** [comment]
Closed via https://github.com/bpmn-io/dmn-js/pull/868
